Week1/Homework/js-exercises/Exercise9.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+let x = 7;
+x = x / 3;
+console.log(x); // expected output: 2.3333333333333335
+let y = 21;
+y = y / 4;
+console.log(y); //expected output: 5.25
+let z = 13;
+z = z / 2;
+console.log(z); //expected output: 6.5

```

Comment:
The operator you are requested to use is the remainder operator `%` not the division operator `/`
So for example the first case will be:
```
let x = 7;
x = x % 3;
console.log(x); // Expected output is 1, which is the remainder of dividing 7 by 3
```